src/main/java/org/benetech/servicenet/service/dto/OrganizationMatchDTO.java

```diff
@@ -137,11 +56,8 @@ public int hashCode() {
     public String toString() {
```

Comment:
please remove toString since @ Data = @ getter, @ Setter, @ tostring, @ EqualsAndHashCode and @ RequiredArgsConstructor